### DISCUSSION TITLE
Implementing the new App Version 3.3.1 funcionality 1phase /3phase /auto

### DISCUSSION THREAD

**Discussion by Matze-aaa** [CJNE/ha-myenergi#512] [Ideas]
With the New myenergi App Version 3.3.1 it is possible to switch manually between 1-/3-phasig charging and auto Mode.
It is guilty for eco and eco+.
It would be very nice to have this implemented in the repository ☺️
Many thanks in advance!

**Comment by heikone** [comment]
Good and helpful observation!
Refer to #509. [Johnny](https://github.com/jwillemsen) already proposed that.
Greetings, Heiko
